src/adagrams.js

```diff
@@ -1,15 +1,106 @@
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  const ENGLISH_LETTERS = generateAlphabet();
+  const LETTERS_FREQUENCY = [
+    9, 2, 2, 4, 12, 2, 3, 2, 9, 1, 1, 4, 2, 6, 8, 2, 1, 6, 4, 6, 4, 2, 2, 1, 2,
+    1,
+  ];
+  const NUM_OF_DRAW_LETTERS = 10;
+
+  const lettersInHand = [];
+  let randomIndex;
+  while (lettersInHand.length < NUM_OF_DRAW_LETTERS) {
+    randomIndex = Math.floor(Math.random() * ENGLISH_LETTERS.length);
```

Comment:
Using the length of ENGLISH_LETTERS to get a random index means that we will only get a random number between 1 and 26 when what we need is a random number between 1 and 98 (which is the total number of letters available). With your current implementation, that means I have a 1/26 chance to get an 'A' and 1/26 change to draw a 'Z'. 

However, the correct distribution should be that I have a 9/98 chance of getting an 'A' and a 1/98 chance of getting a 'Z'. 

If you create a list of letters that reflect the distribution, it would be like [A, A, A, A, A, A, A, A, A, B, B, C, C..... Z] then you can use the length of this list to get a randomIndex